src/main/java/com/aws/greengrass/shadowmanager/model/configuration/ShadowSyncConfiguration.java

```diff
@@ -89,14 +89,28 @@ public static ShadowSyncConfiguration processConfiguration(Map<String, Object> c
      * @param syncConfigurationSet the sync configuration list to add the device thing configuration to.
      * @throws InvalidRequestParametersException if the named shadow validation fails.
      */
-    private static void processOtherThingConfigurations(Map<String, Object> configTopicsPojo, String thingName,
+    private static void processOtherThingConfigurations(Map<String, Object> configTopicsPojo,
```

Comment:
can you add a warning log if both values are specified? We want to encourage using the map

also add a code comment here that config can come in both a map or list  and that the map is preferred so that it is easier to add/remove sync configuration